docs/windows.md

```diff
@@ -0,0 +1,60 @@
+# Experimental Windows support
+
+As from `v0.13.0-rc2`, Buildkit now has exprimental support on Windows. Both `buildctl.exe` and `buildkitd.exe` binaries are being released for testing purposes.
+
+We will apprecate any feedback by [opening an issue here](https://github.com/moby/buildkit/issues/new), as we stabilize the product, especially `buildkitd.exe`.
+
+## Quick start guide
+
+### Platform requirements
+
+- Architecture: `amd64`, `arm64` (binaries available but not officially tested yet).
+- Supported OS: Windows Server 2022, Windows 11.
```

Comment:
We should probably have this as a table indicating OS Versions and Supported Container OS across each version

Reply:
Sure. Linking the main compatibility matrix table from MS docs - https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2019%2Cwindows-11#windows-server-host-os-compatibility